tests/playwright/src/model/pages/agent-workspaces-page.ts

```
@@ -48,6 +49,26 @@ export class AgentWorkspacesPage extends BaseTablePage {
     await expect(this.heading).toBeVisible({ timeout: TIMEOUTS.SHORT });
   }
 
+  private getStatCardValue(label: string): Locator {
+    return this.content.getByText(label, { exact: true }).locator('..').locator('span').first();
+  }
+
+  async getStatCardCount(label: 'Active Sessions' | 'Total Sessions' | 'Configured Agents'): Promise<number> {
+    const valueSpan = this.getStatCardValue(label);
+    await expect(valueSpan).toBeVisible();
+    const text = await valueSpan.textContent();
+    return parseInt(text ?? '0', 10);
```

Biome (2.4.15)

[error] Use Number.parseInt instead of the equivalent global.

(lint/style/useNumberNamespace)